### PR TITLE
Add az-formdata rule to suggest check on formData parameters

### PR DESCRIPTION
**Pull request by mikekistler**
Another simple rule, level "info", to flag formData parameters to check for proper use.

### CODE REVIEW

File: docs/azure-ruleset.md
Comment (markweitzel) [inline docs/azure-ruleset.md:56]:
```suggestion
But it is usually unnecessary and can be overly complicated to use formData when all you are doing is sending a single file-type parameter. 
```

File: docs/azure-ruleset.md
Comment (markweitzel) [inline docs/azure-ruleset.md:57]:
```suggestion
Instead, consider defining a `body` parameter with `type: string, format: binary` and use content-type `application/octet-stream`.
```

**Review by markweitzel — APPROVED** [review]
Overall looks good. Because this is `info` I softened the language a bit--up to you if you want to take the suggestions. No worries if not.

**Comment by mikekistler** [comment]
Sorry I missed your suggested changes before hitting the merge button.  I'll incorporate these into my next PR.

**Pull request merged**
